helm/fdb-operator/templates/manager/secret.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.webhook.secret.name }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Are we doing anything with this webhook secret? I think it came from the defaults in kubebuilder, but we don't have any webhooks right now. We may add some in the future, though.

Reply:
I agree, I just tried to make helm chart of whatever was already present, if we dont use this, I think it can be removed.